extensions/3DTILES_tile_metadata/README.md

```diff
@@ -0,0 +1,161 @@
+# 3DTILES_tile_metadata
```

Comment:
There should be padding rules like there are for batch table binary properties: https://github.com/CesiumGS/3d-tiles/tree/master/specification/TileFormats/BatchTable#padding